packages/frontend-2/components/settings/Dialog.vue

```diff
@@ -55,6 +55,9 @@
               :title="workspaceItem.name"
               collapsible
             >
+              <template #title-icon>
+                <UserAvatar :logo="workspaceItem.logo" size="sm" />
```

Comment:
This component can be reused with one minor change to also work for workspace avatar, `UserAvatar` is a bit misleading in that sense, but also dont know if it's worth to update the naming across all components (and types/composables etc)